### PR TITLE
modify logic of getting package folder path

### PR DESCRIPTION
**Pull request by Ruthenic**
Dev Branch Edition:tm:  
  
don't merge until i fix the TODO

### CODE REVIEW

**Comment by Ruthenic** [comment]
image

**Comment by Ruthenic** [comment]
just tested, and it works

File: src/core/packageLoader/getPackagesPath.ts [hunk 1]
```diff
@@ -16,8 +16,9 @@ switch (processLocation()) {
 function getPackagesPath(): string {
 	switch (processLocation()) {
 		case "MAIN":
-			const origPath = kernelPath = [__dirname, "..", "..", ".."]; //back out of ASAR
-			
+			const origPath = [__dirname, "..", "..", ".."]; //back out of ASAR
+			const kernelPath = origPath
```
Comment (Kyza) [inline src/core/packageLoader/getPackagesPath.ts:20]: Missing semi-colon. Unacceptable.

File: src/core/packageLoader/getPackagesPath.ts [hunk 2]
```diff
@@ -25,10 +25,16 @@ function getPackagesPath(): string {
...
 			
-			// TODO: when i'm on a better code editor restore automatic package folder creation
+			let packagesPath = path.resolve(...kernelPath, "packages");
+			
+			if(!fs.existsSync(packagesPath)) {
+				packagesPath = path.resolve(...origPath, "packages");
+				console.log(
+					`No package directory found. Creating one at "${packagesPath}"`
+				);
+				fs.mkdirSync(packagesPath)
```
Comment (Kyza) [inline src/core/packageLoader/getPackagesPath.ts:37]: Disgrace.

**Review by Kyza — CHANGES_REQUESTED** [review]

**Pull request merged**